.github/workflows/dependency-update.yml

```diff
@@ -18,6 +18,10 @@ env:
   EXAM_EXERCISE_GROUP_ID: "6"
   EXAM_PROGRAMMING_EXERCISE_ID: "48"
 
+concurrency:
+  group: artemis-test-system
+  cancel-in-progress: true
```

Comment:
Does cancel in progress is really a good idea? I'd say they shall simply wait for each other, or am I wrong here?

Reply:
I figured it would be annoying to always wait for the CI when pushing in quick succession. The setup time between CI runs should be high enough to prevent race conditions, ig. What do you think?

Reply:
Doesn't that cancel workflows from dependabot as well ? That would be an issue :)

Reply:
Good point, I've removed it